index.js

```diff
@@ -1,14 +1,24 @@
+const logger = require('./src/logger');
```

Comment:
You need to require the file after the NODE_ENV has been set. So inside the handler.